### DISCUSSION TITLE
[Bug]: iphone & ipad safari problem

### DISCUSSION THREAD

**Discussion by jeromeleong** [danny-avila/LibreChat#4050] [Troubleshooting]
### What happened?

When I use Librichat on my iPad and iPhone, the following error appears in the image.

### Steps to Reproduce

1. use iphone or ipad open Librichat on self host(librechat demo have same promble)
2. try to chat with any model
3. error

### Screenshots

![WechatIMG129](https://github.com/user-attachments/assets/8432b632-2ff2-4aa9-9369-9b8a9fe6d94a)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
Interesting, can you provide more details? iOS version, safari version, etc.? Does it happen as soon as you start the app?

Could be related to this:
https://stackoverflow.com/questions/51568821/works-in-chrome-but-breaks-in-safari-invalid-regular-expression-invalid-group

**Comment by jeromeleong** [comment]
my ios version is 16.1.1 and this is my iphone's screen recording

https://github.com/user-attachments/assets/d4e37ecb-7f58-44e1-b9b2-13952523c897

**Reply by danny-avila** [reply]
I see, thank you for recording this. It gives me an idea what could be the issue.

**Reply by danny-avila** [reply]
can you try disabling LaTeX parsing to see if this works for you? 

![image](https://github.com/user-attachments/assets/ad4cdb7e-1be3-43b2-814a-156ab6dadf4a)

**Reply by jeromeleong** [reply]
i just try to disable that, it don't work. And i find more detail, this error also appears in the chat history, and I am unable to open any chat records in IOS.

**Reply by danny-avila** [reply]
Can you confirm if it works on a different browser, chrome, firefox, something else?

**Reply by jeromeleong** [reply]
Chrome and Firefox both failed to work, maybe it is webkit problem?

**Reply by fuegovic** [reply]
For what it's worth, I can't reproduce that issue on my iPad on iOS 16.7.8

**Reply by christopher-owen** [reply]
Same issue as @jeromeleong

iPhone 7 iOS 15.8.3, using Safari and Firefox browsers.

I appreciate this is an 'old' phone, but just sharing information in case it helps.

**Reply by krimac** [reply]
Hi, same issue, local selfhosted instance, when accesing via browser from iPhone 14 iOS 16.3.1, used firefox, chrome, opera, safari, same issue on all browser from iPhone.

**Comment by danny-avila** [comment]
Just confirmed this is an issue caused by a dependency for iOS < v16.4.

I will see what I can do here

**Reply by danny-avila** [reply]
Let me know if after this PR is merged, if the issue can still be observed: https://github.com/danny-avila/LibreChat/pull/4574
